content/applications/sales/sales/send_quotations/pdf_quote_builder.rst

```diff
@@ -0,0 +1,203 @@
+configured.
+
+To do that, navigate to :menuselection:`Sales app --> Configuration --> Settings`. Then, on the
+:guilabel:`Settings` page, scroll to the :guilabel:`Quotations & Orders` section, and locate the
+:guilabel:`PDF Quote builder` feature.
+
+.. image:: pdf_quote_builder/pdf-quote-builder-feature.png
+   :align: center
+   :alt: The PDF Quote Builder feature located on the Settings page of the Sales application.
+
+Here, custom :guilabel:`Header pages` and :guilabel:`Footer pages` can be uploaded. To upload
+either, click the :guilabel:`Upload your file` button, or the :guilabel:`🖉 (pencil)` icon to the
+right of the desired field, and proceed to locate, select, and upload the desired PDF file from the
+computer's hard drive.
+
+Clicking the :guilabel:`🗑️ (trash)` icon deletes the current PDF file, and replaces the blank field
+with an :guilabel:`Upload your file` button. When :guilabel:`Upload your file` is clicked, proceed
+to upload the desired PDF files, using the same process detailed above.
+
+Once the desired PDF file(s) are uploaded in the appropriate fields in the :guilabel:`PDF Quote
+builder` section of the *Sales* :guilabel:`Settings` page, be sure to click :guilabel:`Save`.
+
+The files uploaded here will be the default PDF used for all quotes.
+
+.. note::
+    Values set here are company-specific.
+
+Dynamic text in PDFs
+====================
+
+While creating custom PDFs for quotes, using any standard PDF editor software, certain dynamic texts
+should be utilized throughout the design of the PDF to ensure Odoo can automatically fill in the
+appropriate content accordingly.
+
+Dynamic text values are, essentially, coded placeholders that Odoo fills in automatically, according
+to the specific information related to the quote.
+
+Dyanmic text values
+-------------------
+
+Below are the dynamic text values and what they will represent in the custom PDFs:
+
+- :guilabel:`name`: Order Reference
+- :guilabel:`partner_id_name`: Customer Name
+- :guilabel:`user_id_name`: Salesperson Name
+- :guilabel:`amount_untaxed`: Untaxed Amount
+- :guilabel:`amount_total`: Total Amount
+- :guilabel:`delivery_date`: Delivery Date
+- :guilabel:`validity_date`: Expiration Date
+- :guilabel:`client_order_ref`: Customer Reference
+
+Once the PDF file(s) are complete, save them to the computer's hard drive, and proceed to upload
+them to Odoo, via :menuselection:`Sales app --> Configuration --> Settings --> PDF Quote builder -->
+Save`.
+
+Add PDF to product
+==================
+
+In Odoo *Sales*, it's also possible to add a custom PDF to a product. When a PDF is added to a
+product, and that product is used in a quotation, that PDF is also inserted in the final PDF.
+
+To add a custom PDF to a product, start by navigating to :menuselection:`Sales app --> Products -->
+Products`, and selected the desired product to which a custom PDF should be added.
+
```

Comment:
A document could also be added to a product variant instead of a product. Be aware that if there are documents on a product AND on its variant, only the documents in the variant will be shown. =]

Reply:
Thanks for the reminder. I added this as a note admonition right below this. 👍